cmd/plugin/builder/template/plugintemplates/plugin-tooling.mk.tmpl

```diff
@@ -36,7 +36,7 @@ PLUGIN_DEBUG=false
 endif
 
 # Add supported OS-ARCHITECTURE combinations here
-PLUGIN_BUILD_OS_ARCH ?= linux-amd64 windows-amd64 darwin-amd64 darwin-arm64 linux-arm64
+PLUGIN_BUILD_OS_ARCH ?= linux-amd64 windows-amd64 darwin-amd64 linux-arm64 darwin-arm64 windows-arm64
```

Comment:
Considering we are adding the `linux-arm64` as well, should we mention `linux-arm64` in the PR description as well?

Reply:
Sorry for the confusion, I just reordered the list so it was consistent: `linux-arm64` was already there, just in a different order.